string_scrap.js

```diff
@@ -1,31 +1,43 @@
+//1
 let word = 'PURSUIT';
 console.log(word[0]);
 
+//2
 let str = 'PURSUIT';
 console.log(str.length);
 
+//3
 let name = 'PURSUIT';
 let lastCHar = name.charAt(name.length - 1);
 console.log(lastCHar);
 
+//4
 let name1 = 'jimmy';
 let lastCHar2 = name1.charAt(name1.length - 1);
 console.log(lastCHar2);
 
+//5
 let drEvil = '10';
 if (drEvil[5] === '0' || drEvil [2] === '0') {
   console.log('pinky');
 } else {
   console.log('10 dollars');
 }
 
+//6
 let strc = 'train';
 let stru = 'box';
 if (strc = 'box' || 'train') {
```

Comment:
This is wrong syntax. Correct way to use || operator inside an if statement is as following: 
if(strc === "box" || strc === "train") {
}